subo/util/token.go

```diff
@@ -2,36 +2,44 @@ package util
 
 import (
 	"io/ioutil"
-	"os"
 	"path/filepath"
 
 	"github.com/pkg/errors"
 )
 
-func getTokenTmpDir() string {
-	tokenPath := filepath.Join(os.TempDir(), "suborbital", "compute", "envtoken")
-	return tokenPath
+func getTokenPath() (string, error) {
+	tokenPath, err := CacheDir("compute")
+	if err != nil {
+		return "", err
+	}
+
+	return filepath.Join(tokenPath, "envtoken"), nil
 }
 
 func WriteEnvironmentToken(tokenStr string) error {
-	tokenPath := getTokenTmpDir()
-	if _, err := os.Stat(tokenPath); os.IsNotExist(err) {
-		if err := os.MkdirAll(filepath.Dir(tokenPath), PermDirectoryPrivate); err != nil {
-			return errors.Wrap(err, "failed to Mkdir")
-		}
+	tokenPath, err := getTokenPath()
+	if err != nil {
+		return err
 	}
 
 	if err := ioutil.WriteFile(tokenPath, []byte(tokenStr), PermFilePrivate); err != nil {
-		return errors.Wrap(err, "failed to WriteFile for token")
+		return errors.Wrapf(err, "failed to write %s "+tokenPath)
```

Comment:
this will error because there's a placeholder (`%s`), but there's nothing going in there. That plus needs to be comma :)
```suggestion
		return errors.Wrapf(err, "failed to write %s ", tokenPath)
```